e2e/tests/apiproduct-overview-tab.spec.ts

```
@@ -0,0 +1,335 @@
+    await expect(page.locator('dd').filter({ has: page.locator('text=manual') })).toBeVisible();
+  });
+
+  test('should display authentication methods with "Not set" and configured states', async ({
+    page,
+  }) => {
+    // Test "Not set" state
+    await navigateToAPIProductOverview(page, TEST_NAMESPACE, 'test-product-no-auth');
+    await expect(page.locator('text=About').first()).toBeVisible({ timeout: 10_000 });
+
+    await expect(page.locator('dt:has-text("Authentication Methods")')).toBeVisible();
+    let authMethodsValue = page
```

ESLint

[error] 'authMethodsValue' is never reassigned. Use 'const' instead.

(prefer-const)